standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2070,18 +2096,30 @@ struct CreateDatabaseRequest {
   8: optional string catalogName,
   9: optional i32 createTime,
   10: optional string managedLocationUri,
-  11: optional string type,
-  12: optional string dataConnectorName
+  11: optional DatabaseType type,
+  12: optional string dataConnectorName,
+  13: optional string remote_dbname
```

Comment:
why do we need the remote_dbname in this? this will be provided in the parameters along with the createDB request. 
Also, I am a bit concerned about changing type from string to DatabaseType. its cleaner but is there another benefit?

Reply:
I'm trying to point to create_database_req(req) instead of create_database(database), by converting the database object to a request object. As a result, I had to introduce whatever is in the database object to the CreateDatabaseRequest object. If this is not a good change, what do you suggest I should be doing?